### PR TITLE
Patching react-native-code-push for linking assets

### PR DESCRIPTION
**Pull request by ferrannp**
Fixes https://github.com/kiwicom/mobile/issues/1274.

### CODE REVIEW

File: android/app/build.gradle [hunk 1]
```diff
@@ -155,6 +155,7 @@ android {
 }
 
 dependencies {
+    compile project(':react-native-code-push')
```
Comment (ferrannp) [inline android/app/build.gradle]: In this way it will detect it's already "linked" even if `app` does not use it.

Comment (tbergq) [reply android/app/build.gradle]: shouldn't it be `implemenation` and not `compile`?

File: ios/reactNativeApp.xcodeproj/project.pbxproj [hunk 2]
```diff
@@ -5,7 +5,6 @@
 	};
 	objectVersion = 46;
 	objects = {
-
```
Comment (tbergq) [inline ios/reactNativeApp.xcodeproj/project.pbxproj]: Just out of curiosity, why do these changes happen from time to time?

Comment (ferrannp) [reply ios/reactNativeApp.xcodeproj/project.pbxproj]: Yeah I don't really know, it looks it differs depending who makes them... I guess we can investigate it in the future but yeah...

File: android/app/build.gradle (hunk 1 above)
Comment (ferrannp) [reply android/app/build.gradle]: Yeah but link puts `compile` still. I can change it manually however as this is the `app` it does not really matter.

Comment (tbergq) [reply android/app/build.gradle]: Ok, then I'll allow it 🙂

**Pull request merged**